src/utils/logging.jl

```diff
@@ -1,6 +1,14 @@
 
 import Logging
 
+# These use PascalCase to avoid clashing with source filenames.
+const LOG_GROUP_RECORDER = :Recorder
+const LOG_GROUP_SERIALIZATION = :Serialization
+const LOG_GROUP_SYSTEM = :System
+const LOG_GROUP_TIME_SERIES = :TimeSeries
+const LOG_GROUPS =
+    (LOG_GROUP_RECORDER, LOG_GROUP_SERIALIZATION, LOG_GROUP_SYSTEM, LOG_GROUP_TIME_SERIES)
+
```

Comment:
Where is `LOG_GROUPS` being used? I couldn't find another instance of it using grep.

Reply:
> Where is `LOG_GROUPS` being used? I couldn't find another instance of it using grep.

I started making a "disable all predefined" option but then abandoned it. This could still be convenient if you don't remember the formal names or spellings.